src/main/resources/templates/swift/exercise/Sources/${packageNameFolder}Lib/Client.swift

```diff
@@ -0,0 +1,86 @@
+﻿import Foundation
+
+public class Client {
+
+	// TODO: Implement BubbleSort
```

Comment:
Minor nit: I assume this wants the same 4-space indentation like the following `TODO`s.

Reply:
keen eye!